src/app/common/button/button.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="button" #button
+     [ngStyle]="styles.elStyle"
+     (mouseover)="onHover(button, true)"
```

Comment:
nije mi najjasnije ni dodavati hover klase pomocu javasctripta?? Jel imas neki poseban razlog za to?